conda-forge.yml

```diff
@@ -1,8 +1,14 @@
-bot: {automerge: true}
-build_platform: {osx_arm64: osx_64}
+bot:
+  automerge: true
+build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
```

Comment:
```suggestion
```

In the meantime, I suggest to revert this.

Reply:
We would like to figure this out. Kicking it down the road doesn't change the fact we will need to figure it out. So we might as well do it here

Reply:
> Kicking it down the road doesn't change the fact we will need to figure it out. So we might as well do it here

I think this is conflating two things: the pypy migration, and cross-compiling a package against pypy. Has the latter been done anywhere already? I'm not saying this isn't worth fixing (and soon), I just don't see why we _need_ to couple the migration to forcefully cross-compiling everything.

Reply:
Agree with @jakirkham

>  and cross-compiling a package against pypy. Has the latter been done anywhere already? 

Yes, lots of feedstocks including numpy and scipy.

Reply:
OK, I had misremembered that the [switch](https://github.com/conda-forge/scipy-feedstock/pull/193) to cross-compilation had come after dropping pypy.

Let's fix it then! 🙃